python/lsst/sconsUtils/state.py

```diff
@@ -215,6 +215,14 @@ def _initEnvironment():
       CC_LOGGER_BIN
     """.split()
 
+    implicitMultithreadingVars = """
```

Comment:
Please put a comment here about why you set these and how you found them.

Reply:
I have updated the list of variables to be more complete from lsst.utils even covering less common env vars.